Resources/Prototypes/_Pirate/Access/interdyne.yml

```
@@ -0,0 +1,11 @@
+- type: accessLevel
+  id: Interdyne
+  name: id-card-access-level-Interdyne
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)